roles/edpm_deploy/tasks/main.yml

```diff
@@ -162,7 +164,38 @@
         cmd: >-
           oc wait OpenStackDataPlaneNodeSet {{ cr_name }}
           --namespace={{ cifmw_install_yamls_defaults['NAMESPACE'] }}
-          --for=condition=ready
+          --for=condition=SetupReady
+          --timeout={{ cifmw_edpm_deploy_timeout }}m
+
+    - name: Apply DataPlaneDeployment resource
+      kubernetes.core.k8s:
+        kubeconfig: "{{ cifmw_openshift_kubeconfig  }}"
+        api_key: "{{ cifmw_openshift_token | default(omit) }}"
+        context: "{{ cifmw_openshift_context | default(omit) }}"
+        state: present
+        definition: "{{ lookup('file', cifmw_edpm_deploy_crs_kustomize_result.output_path) | from_yaml_all | selectattr('kind', 'search', cifmw_edpm_deploy_step2_kind) }}"
+
+    - name: Wait for OpenStackDataPlaneDeployment become Ready
+      when: not cifmw_edpm_deploy_dryrun
```

Comment:
The entire block is conditioned to the dry-run flag, this, and all the occurrences of the flag inside the block, is not needed.

Reply:
True, I can improve that in a next PR, which should include this change for baremetal deploy too.